.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thank you for submitting a PR to the rust rpm implementation!
+-->
+
+# PR Content Desc
```

Comment:
Let's splash out and spend the extra few bytes to make this "Description" 🙂

Reply:
I'll take care of it, it'd be great if you could add suggestions for the edits in future PRs, so I can fix it instantly

Reply:
```suggestion
# PR Content Description
```